kerckhoff/packages/serializers.py

```diff
@@ -172,3 +174,82 @@ def get_version_data(self, obj: Package):
     class Meta(PackageSerializer.Meta):
         fields = PackageSerializer.Meta.fields + ("version_data",)
         read_only_fields = PackageSerializer.Meta.read_only_fields + ("version_data",)
+
+class PackageInfoSerializer(serializers.ModelSerializer):
+    package_set = serializers.StringRelatedField()
+    latest_version = serializers.StringRelatedField()
+    description = serializers.SerializerMethodField()
+    folder_id = serializers.SerializerMethodField()
+    folder_url = serializers.SerializerMethodField()
+    metadata = serializers.SerializerMethodField()
+    data = serializers.SerializerMethodField()
+    cached_article_preview = serializers.SerializerMethodField()
+
+    def get_description(self, obj: Package):
+        obj.fetch_cache()
+        versionSerializer = PackageVersionSerializer(obj.get_version(obj.latest_version.id_num))
+        return versionSerializer.data["version_description"]
+
+    def get_folder_id(self, obj: Package):
+      return obj.metadata["google_drive"]["folder_id"]
+    
+    def get_folder_url(self, obj: Package):
+      return obj.metadata["google_drive"]["folder_url"]
+
+    def get_metadata(self, obj: Package):
+      return {}
+    
+    def get_data(self, obj: Package):
+        obj.fetch_cache()
+        package_items = obj.get_version(obj.latest_version.id_num).packageitem_set.all() 
+        img_urls = {}
+        supported_image_types = [".jpg", ".jpeg", ".png"]
+        for file in package_items:
+            file_ext = os.path.splitext(file.file_name)[-1]
+            if(file.file_name == "article.aml"):
+                aml_data = file.data["content_rich"]["data"]
+            if(file_ext in supported_image_types):
```

Comment:
You don't seem to have included the images into the JSON data. I think you should either get rid of this code, or just put them into the JSON as well.

Reply:
I kept these code because I guessed we would include images somewhere later, but I just deleted these for now as you asked.